templates/templates/base_html.template

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
ditto on moving to a subdir for preview templates

Reply:
changed